loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -231,12 +232,12 @@ L.TileSectionManager = L.Class.extend({
 						this._zoomAnimation();
 					}
 				},
-				zoomStepEnd: function () {
+				zoomStepEnd: function (zoom, newCenter) {
```

Comment:
The changes in this file also seems unrelated to "Control.Header: Bring non-shared functions from rows and columns". Either interactive rebase can be used to move this to relevant previous commit or this file's change can be made into a separate commit.

Reply:
My comments on rebasing are relevant if the commit history in this PR is to be maintained when merging to master.